dotnet/src/Connectors/Connectors.Anthropic/Core/Models/AnthropicResponse.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using Microsoft.SemanticKernel.ChatCompletion;
+
+namespace Microsoft.SemanticKernel.Connectors.Anthropic.Core;
+
+internal sealed class AnthropicResponse
+{
+    [JsonRequired]
+    [JsonPropertyName("id")]
+    public string Id { get; init; } = null!;
+
+    [JsonRequired]
+    [JsonPropertyName("type")]
+    public string Type { get; init; } = null!;
+
+    [JsonRequired]
+    [JsonPropertyName("role")]
+    [JsonConverter(typeof(AuthorRoleConverter))]
+    public AuthorRole Role { get; init; }
+
+    [JsonRequired]
+    [JsonPropertyName("content")]
+    public IReadOnlyList<AnthropicContent> Contents { get; init; } = null!;
+
+    [JsonRequired]
+    [JsonPropertyName("model")]
+    public string ModelId { get; init; } = null!;
+
+    [JsonPropertyName("stop_reason")]
+    public AnthropicFinishReason? FinishReason { get; init; }
```

Comment:
Missing XmlDocs for all public.

Suggest for internals to use the same language as the provider, try also making the properties internal if possible.

```suggestion
    public AnthropicFinishReason? StopReason { get; init; }
```

Reply:
Response class is internal, so I omitted xmlDoc.

Reply:
We are adding XmlDoc for internal classes as well for clarity of any connector implementer, see:

https://github.com/microsoft/semantic-kernel/pull/6898#discussion_r1648455626